### PR TITLE
BANNO.CDRENEW.V1.CFG change term option update

### PR DESCRIPTION
**Pull request by tkainz**
Added clarifying description for option #2  - change cert term

### CODE REVIEW

File: certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG [hunk 1]
```diff
@@ -31,6 +33,11 @@
 *   OPTION4: Renew certificate (roll over)
 *   OPTION5: Disburse funds by check
 *   OPTION6: Suspend certificate
+*
+*   OPTION 2, if selected by the member, it results in the member being
```
Comment (jucarson) [inline certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG:37]: Looks good to me, I would just remove the word "it" on line 37 or correct the grammar in another way.

Comment (tkainz) [reply certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG:37]: Yep, I would agree.... but I was asked to specifically add it in so..?   :-)

Comment (jucarson) [reply certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG:37]: alright! :)

**Review by jucarson — APPROVED** [review]

File: certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG (hunk 1 above)
Comment (jordan-keenan) [reply certificate-renewal/LETTERSPECS/BANNO.CDRENEW.V1.CFG:37]: Sorry I got behind. 
Yes; the change looks good 
to let the person setting up the PO config know what Option 2 does,  
as the UX just tells the member to contact the CU to change the certificate term when they select that in Banno.

**Pull request merged**